app/assets/javascripts/discourse/tests/integration/components/plugin-outlet-test.gjs

```diff
@@ -464,7 +465,7 @@ module("Integration | Component | plugin-outlet", function (hooks) {
         <template>
           <PluginOutlet
             @name="test-name"
-            @deprecatedArgs={{hash
+            @deprecatedArgs={{lazyHash
```

Comment:
Should `@deprecatedArgs` in `components/header/contents.gjs` also use `lazyHash`?